app/page-partials/admin-dashboard/AdminTabs/GithubTabContent.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
```

Comment:
I see lot of redundant code between this file and BceidTabContent.tsx. Is it possible to merge these two into one and reuse the code?